sql/core/src/main/scala/org/apache/spark/sql/execution/basicPhysicalOperators.scala

```diff
@@ -91,60 +91,39 @@ case class ProjectExec(projectList: Seq[NamedExpression], child: SparkPlan)
   }
 }
 
-
-/** Physical plan for Filter. */
-case class FilterExec(condition: Expression, child: SparkPlan)
```

Comment:
@maropu  I don't understand why we need change `FilterExec` ?

Reply:
Just for sharing code to process predicates between aggregates and filters.